engine/src/test/java/io/camunda/zeebe/engine/state/migration/DbMigratorImplTest.java

```diff
@@ -110,7 +215,9 @@ private RunInTransactionContext(final ZeebeDbTransaction transaction) {
     @Override
     public void runInTransaction(final TransactionOperation operations) {
       try {
+        transaction.rollback();
```

Comment:
❓ Why we rollback here?

Reply:
That was necessary to emulate how real transactions behave. It's now removed though because we don't directly interact with transactions anymore.